Intel_SighPP_CMake/Intel_SighPP_CMake/src/interface_controller/output_stream_controller.cpp

```diff
@@ -41,72 +41,88 @@ void OutputStreamController::output_to_depth_window(const rs2::frame& frame) con
 	const cv::Mat depth_mat(cv::Size(width, height), CV_8UC3, const_cast<void*>(frame.get_data()), cv::Mat::AUTO_STEP);
 	imshow(depth_output_window_, depth_mat);
 }
-
-void OutputStreamController::output_to_color_window(cv::Mat color_matrix, cv::Mat depth_matrix, const std::vector<PrioritisedClassificationResult>& vector) const
+void OutputStreamController::output_to_color_window(cv::Mat color_matrix, cv::Mat depth_matrix, const std::vector<ClassificationItem>& vector) const
 {
-	for (auto&& prioritised_classification_result : vector)
+
+	//const auto width = frame.as<rs2::video_frame>().get_width();
+	//const auto height = frame.as<rs2::video_frame>().get_height();
+
+	//const cv::Mat color_mat(cv::Size(width, height), CV_8UC3, const_cast<void*>(frame.get_data()), cv::Mat::AUTO_STEP);
```

Comment:
Do we need to use the three lines that are commented out?

Reply:
Fixed.